shopify/README.md

```diff
@@ -35,29 +35,29 @@ View list of past orders
 ```
 /nc shopify_orders
 ```
-![Shopify orders command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/orders.PNG)
+![Shopify orders command](screenshots/orders.PNG)
 
 Add a note to a past order
 NOTE: order_id can be found using /nc shopify_orders
 Params: order_id (Unique ID of an order), note (Text to attach to order)
 ```
 /nc shopify_add_note <order_id> <note>
 ```
-![Shopify add note command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/addNote.PNG)
+![Shopify add note command](screenshots/addNote.PNG)
 
 View list of products sold
-Params: search_terms (Comma deliminated list of text to filter by) EX: "textone, text two, text three four"
+Params: search_terms (Comma delimited list of text to filter by) EX: "textone, text two, text three four"
 ```
 /nc shopify_products_sold [<search_terms>]
 ```
-![Shopify products sold command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/productsSold.PNG)
+![Shopify products sold command](screenshots/productsSold.PNG)
 
 View product inventory
-Params: search_words (Comma deliminated list of text to filter by) EX: "textone, text two, text three four"
+Params: search_words (Comma delimited list of text to filter by) EX: "textone, text two, text three four"
 ```
 /nc shopify_inventory [<search_words>]
 ```
-![Shopify inventory command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/inventory.PNG)
```

Comment:
The Shopify help command fetches readme directly from GitHub and relative URLs won't resolve on Mattermost.